sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -1144,11 +1036,9 @@ export class KeyClient {
       keyType: keyBundle.key ? keyBundle.key.kty : undefined,
       properties: {
         id: keyBundle.key ? keyBundle.key.kid : undefined,
-        name: parsedId.name,
```

Comment:
why did these disappear?

Reply:
These are not needed! The compiler started complaining recently.